subsys/bluetooth/controller/ll_sw/pdu.h

```diff
@@ -185,6 +185,10 @@
 #define EVENT_CLOCK_JITTER_US   2
 /* Event interframe timings */
 #define EVENT_IFS_US            150
+/* Event interframe timings, low latency mode */
+#define EVENT_IFS_LOW_LAT_US    (CONFIG_BT_CTLR_IFS_LOW_LATENCY)
```

Comment:
What is the rationale behind adding this define that is defined to be the same as a Kconfig, rather than using the Kconfig value directly? 

IMO the latter may provide more information to the reader in terms of knowing that it's a configurable value and not a static value